BigCommerce/UserInterface/01a_Connections.md

```diff
@@ -0,0 +1,198 @@
+
+**Credentials** 
+
+- *Store hash*   
+    Click the field to edit the hash key of the *BigCommerce* store.
+
+- *Access token*   
+    Click the field to edit the access token for the *BigCommerce* connection.
+
+- [SAVE]  
+    Click this button to save any changes made.
+
+
+
+## Edit BigCommerce connection &ndash; Settings
+
+*Omni-Channel > Settings > Tab CONNECTIONS > Select BigCommerce connection > Tab Settings*
+
+![Edit BigCommerce settings](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/EditConnectionSettings.png "[Edit BigCommerce settings]")
+
+The *Settings* tab is composed of the following setting entries:
+- [Variant option type](#variant-option-type)
+- [Order filters](#order-filters)
+
+
+### Variant option type
+
+*Omni-Channel > Settings > Tab CONNECTIONS > Select BigCommerce connection > Tab Settings > Menu entry Variant option type*
+
+![Variant option type](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/VariantOptionType.png "[Variant option type]")
+
+- *Variant option type used for variants*   
+   Use this dropdown list to select the type to select variants. For detailed information, see [Configure BigCommerce connection](../Integration/01_ManageBigCommerceConnection.md#configure-bigcommerce-connection).
+
+
+
+### Order filters
+
+*Omni-Channel > Settings > Tab CONNECTIONS > Select BigCommerce connection > Tab Settings > Menu entry Order filters*
+
+![Order filters](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/OrderFilters.png "[Order filters]")
+
+- *Filter orders to import by*   
+   Enable the toggles at the order status to define the orders to be imported to *Actindo*.  
+   For detailed information on the *BigCommerce* order statuses see the *BigCommerce* documentation under [https://developer.bigcommerce.com/docs/rest-management/orders/order-status#get-all-order-statuses](https://developer.bigcommerce.com/docs/rest-management/orders/order-status#get-all-order-statuses).
+   
+- *Import only orders with an ID greater than*   
+   This option is only relevant if you have orders in *BigCommerce* that should not be imported to *Actindo* in general.   
+   <!----Wenn es bei altem Text bleibt: Enter the ID of the last order you want to keep in *BigCommerce* and you do not want to import to the *Omni-Channel* module.-->
+    Enter the ID of the first order you want to import to the *Omni-Channel* module.
+- *Import only orders created after*   
+   This option is only relevant if you have orders in *BigCommerce* that should not be imported to *Actindo* in general.    
+   <!---Wenn es bei altem Text bleibt:Enter the date of the previous day from which you want to start importing orders to the Omni-Channel module.-->  
+   Enter the date from which you want to import orders into the *Omni-Channel* module.
+
+- *Import archived orders*    
+   By default, this toggle is disabled. Enable this toggle if you want to import orders that have been already archived.
```

Comment:
Hier fehlt der Setting Offers, der in Integration beschrieben ist, oder?

![image](https://github.com/Actindo-AG/Documentation/assets/99268392/8f21fa07-0b60-42d8-ba78-e21eb11e4ba6)

Reply:
Done